src/test/java/com/ibm/cloud/sdk/core/util/DateDeserializerTest.java

```diff
@@ -17,14 +17,13 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.fail;
 
+import java.text.SimpleDateFormat;
```

Comment:
Looks like an intellij user previously messed with this class...  @mkistler any idea who that might have been?  :)